app/uk/gov/hmrc/customsservicestatusfrontend/controllers/BaseFrontendController.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
```

Comment:
minor: as we have just created this file, can we change this to 2024 ?

Reply:
updated - thanks @sureshhmrc